torch/_inductor/kernel/mm_scaled.py

```diff
@@ -553,6 +554,12 @@ def tuned_scaled_mm(
             for config in scaled_mm_configs(m, n, k):
                 if k == 16 and config.kwargs["BLOCK_M"] >= 64:
                     continue  # Triton crashes in this case
+
+                # On NVIDIA B200 GPUs, K dim must be >= 32 for tcgen05.mma.kind::f8f6f4.* PTX instruction to be valid
```

Comment:
I would opt against checking this here.  I believe there are more situations that CUTLASS-based kernel would fail, regarding the input shapes, but also alignment - it would be rather hard to enumerate all these here, but in all cases CUTLASS will report run-time error.  Instead, I would just skip corresponding test case (using [`SM100OrLater`](https://github.com/pytorch/pytorch/blob/d4d7d813fa703a72a583174ca4fdcdb06808a738/torch/testing/_internal/common_cuda.py#L35) check, that is the same as `using_b200`).